site/docs/v1.3.0-beta.1/api-types/README.md

```diff
@@ -0,0 +1,16 @@
+# Table of Contents
+
+## API types
+
+Here we list the API types that have some functionality that you can only configure via json/yaml vs the `velero` cli
+(hooks)
```

Comment:
"(hooks)" seems out of place

Reply:
k, that's existing text in the master/v1.2 docs, so I'm not gonna change it as part of this PR, can do it separately.

Reply:
Maybe I misunderstand what hooks mean. Does it mean like cli commands?

Reply:
we can resolve this in a different PR.